distribution/openrct2.d.ts

```diff
@@ -1253,16 +1499,27 @@ declare global {
         edgeStyle: number;
     }
 
-    // does not support TileModifyType::AnyPaste
+    /**
+     * Does not support `TileModifyType::AnyPaste`
```

Comment:
```suggestion
     * @todo Does not support `TileModifyType::AnyPaste`
```

Would make sense here too, right?